lib/widgets/home/courses_row_item_widget.dart

```diff
@@ -50,7 +50,7 @@ class CoursesRowItemWidget extends StatelessWidget {
     if (data != null) {
       return Text(data.subtitle, style: Theme.of(context).textTheme.subtitle1);
     } else {
-      return LoadingTextBoxWidget(height: 21);
+      return Container(height: 0, width: 0); //LoadingTextBoxWidget(height: 21);
```

Comment:
What's the idea here @harshadmanglani ?

Reply:
Oh, this was just removing the extra `LoadingTextBoxWidget` when the data loads. I sent you a screenshot on Slack, if you remember. Let me resend it.

Reply:
![LoadingTextBox](https://user-images.githubusercontent.com/47568882/134865110-280146dc-dc85-4f14-a2b7-2c0af848e654.png)

Reply:
Left image: original
Right image: changed

Feel free to undo that :) I was just trying something out

Reply:
Ah yeah, looks much better now thanks :)